### PR TITLE
Add travis setup for heroku app and repo

### LINKED ISSUE #2: Deploy backend
deploy api endpoints to heroku

### PR DESCRIPTION
**Pull request by amsmyth1**
### What Changed?
- Initial setup of Travis CI, created .travis.yml
- Setup Heroku app

### Closed Issues
Close #2 #1 

### What's Next?
- Confirm travis ci is pushing to heroku

### Known Issues?
- none

### Test Coverage?
- n/a

### README & Documentation
- TO DO: Add setup instructions to the readme, link to the heroku app

### CODE REVIEW

**Review by klatour324 — APPROVED** [review]
Thanks for getting this initially set up! I've more recently been working with Circle CI and don't recall with Travis CI if we all need to be added as collaborators?

File: Gemfile [hunk 1]
```diff
@@ -24,13 +24,25 @@ gem 'puma', '~> 3.11'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
+gem 'travis'
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
```
Comment (percworld) [inline Gemfile:29]: interesting, I thought it was only 'cors'

File: Gemfile.lock [hunk 2]
```diff
@@ -106,7 +174,40 @@ GEM
     rb-fsevent (0.11.0)
```
Comment (percworld) [inline Gemfile.lock:174]: I would really like to know more about fsevents if you wonderful back-enders know something about this.

**Review by percworld — COMMENTED** [review]
Nice work on travis.yml, I'm just peeking to get a sense of how this builds, great work today!

**Comment by amsmyth1** [comment]
> Thanks for getting this initially set up! I've more recently been working with Circle CI and don't recall with Travis CI if we all need to be added as collaborators?

I believe we all should have access since we are Organization owners for QuickCommand. Let me know if you cannot get access to the builds

File: Gemfile.lock (hunk 2 above)
Comment (amsmyth1) [reply Gemfile.lock:174]: The Gemfile.lock is kind of a magical document that pulls in all the dependencies for the gems declared in the Gemfile. It also makes sure all the same gem versions are utilized by each user when they pull the repo to their remote (and run bundle install). I am do not know much about rb-fsevent

**Review by jmichlitsch — APPROVED** [review]
Looks good for the initial setup, still need to personally look more into CORS but otherwise looks great

**Pull request merged**